cmd/raidz_test/raidz_test.c

```diff
@@ -119,7 +120,7 @@ static void usage(boolean_t requested)
 	    "\t[-B benchmark all raidz implementations]\n"
 	    "\t[-e use expanded raidz map (default: %s)]\n"
 	    "\t[-r expanded raidz map reflow offset (default: %llx)]\n"
-	    "\t[-v increase verbosity (default: %zu)]\n"
+	    "\t[-v increase verbosity (default: %u)]\n"
```

Comment:
`enum` is an `int` normally.  I'd use `%i` (or `%d`, depending on your preference), no?

Reply:
I mean, it doesn't really matter, it's gonna be like five max and passing ints to %u is fine and it's used as-if it's unsigned but sure.